lib/plugins/console/generate.js

```diff
@@ -44,11 +46,12 @@ function generateConsole(args) {
   function writeFile(path, force) {
     var dest = join(publicDir, path);
     var cacheId = 'public/' + path;
+    var dataStream = wrapDataStream(route.get(path), {ignoreErrors: !bail});
```

Comment:
consider simplify the `wrapDataStream` to 
```
wrapDataStream(path, { bail })
```
so we don't have to remember the relationship between `ignoreErrors` and `bail`.

Reply:
makes sense 👍